pythonvectordbceph.py

```diff
@@ -84,14 +84,14 @@ def pythonvectordbappceph():
     # TODO: parse other metadatas and add to the collection
     event_data = json.loads(request.data)
     object_key = event_data['Records'][0]['s3']['object']['key']
+    event_type = event_data['Records'][0]['eventName']
     app.logger.debug(object_key)
 
     # Create collection which includes the id, object url, and embedded vector
     if not client.has_collection(collection_name=collection_name):
         fields = [
-                FieldSchema(name='id', dtype=DataType.INT64, is_primary=True, auto_id=True),
```

Comment:
why did you remove the "id" field?

Reply:
Initially  thought we need integer to have primary key. But later I found that it can be varchar as well. For the upsert(update operation) the primary key is must since primary key is `auto id`, we may require additional operation to fetch primary key. So by setting the object url primary key this can be avoided. Currently I have primary key as object url, if it make sense to use someother value like Etag, we can change accoridingly